Makefile

```
@@ -0,0 +1,266 @@
+run: build
+	@echo "Running Trndi from $(OUTDIR)"
+	@set -e; \
+	if [ -x "$(OUTDIR)/Trndi" ]; then "$(OUTDIR)/Trndi" $(RUN_ARGS); \
+	elif [ -x "$(OUTDIR)/Trndi.app/Contents/MacOS/Trndi" ]; then "$(OUTDIR)/Trndi.app/Contents/MacOS/Trndi" $(RUN_ARGS); \
+	elif [ -f "$(OUTDIR)/Trndi.exe" ]; then "$(OUTDIR)/Trndi.exe" $(RUN_ARGS); \
+	elif [ -x "./Trndi" ]; then ./Trndi $(RUN_ARGS); \
+	elif [ -f "./Trndi.exe" ]; then ./Trndi.exe $(RUN_ARGS); \
+	else echo "Executable not found in $(OUTDIR) or project dir; build first"; exit 1; fi
+
+# Build without extensions (use a temporary copy of the .lpi so original is untouched)
+noext: check
```

checkmake (0.2.2)

[warning] Target body for "noext" exceeds allowed length of 5 (36).

(maxbodylength)